### DISCUSSION TITLE
crossfade skip files less than 5 seconds

### DISCUSSION THREAD

**Discussion by caf2022** [savonet/liquidsoap#2388] [User Support]
How to skip the crossfade for playlist files that have less then 5 seconds duration?

**Comment by toots** [comment]
Hi,

What pipeline do you have to queue your files? This might be something that you want to treat in the pre-processing phase, i.e. by pre-computing file duration and tagging them to be treated specifically by your crossfade operator.

**Comment by caf2022** [comment]
I'm using very simple code:

```
playlist = playlist(reload_mode="watch", loop=true, mode="normal", "/home/playlist.pls")
playlist = crossfade(duration=2.00, fade_out=2.00, fade_in=1.00, playlist )
output.shoutcast(%fdkaac(bitrate=128,.....
```
